flocx_market/tests/unit/api/test_app_bid.py

```diff
@@ -38,12 +38,12 @@ def test_get_bids(mock_get_all, client):
     test_result = [test_bid_1, test_bid_2]
     mock_get_all.return_value = test_result
     response = client.get("/bid", follow_redirects=True)
-    assert response.status_code == 200
-    assert len(response.json) == 2
-    assert any(x['creator_id'] == test_bid_1.creator_id
-               for x in response.json)
-    assert any(x['creator_id'] == test_bid_2.creator_id
-               for x in response.json)
+    assert response.status_code == 404
```

Comment:
Does this test not work?

Reply:
This is the one that sends get request with None bid_id.

Reply:
Ah, I think there's some misunderstanding - I'll comment above.